### PR TITLE
add get_repos_contributed fxn, #25

### PR DESCRIPTION
**Pull request by jsta**
See #25

### CODE REVIEW

**Comment by maelle** [comment]
Thank you! I made a few comments/suggestions.

**Comment by jsta** [comment]
Did you save them? I don't see any comments...

**Comment by maelle** [comment]
I think you should be able to see them by scrolling up? They are in the files but appear in this thread.

**Comment by maelle** [comment]
oops only I could see them because the review hadn't been submitted, sorry about that!!

File: R/get_contributions.R [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#' Get contributions by a given account
```
[the remark this answers is not in the document]
Comment (jsta) [reply R/get_contributions.R]: :+1:

File: R/get_contributions.R [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#'
+#' @inheritParams get_repos
+#'
+#' @return tibble with name (owner/repo), description, primary language, stargazer count, is_fork, is_privatem, and is_created.
```
[the remark this answers is not in the document]
Comment (jsta) [reply R/get_contributions.R]: :+1:

File: R/get_contributions.R [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+#' @export
+#'
+#' @examples \dontrun{
+#' cb <- get_contributions()
+#' }
+get_contributions <- function(privacy = "PUBLIC"){
```
[the remark this answers is not in the document]
Comment (jsta) [reply R/get_contributions.R]: I agree but I'm having some trouble figuring out how to do this.

**Comment by jsta** [comment]
OK, I think I figured out how to specify a user in 4c840a1

**Comment by maelle** [comment]
Thank you!

**Pull request merged**